common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2670,8 +2670,9 @@ StScrollBar {
 .unlock-dialog-notification-count-text {
   font-weight: bold;
   padding: 0 6px;
-  color: $osd_bg_color;
-  background-color: transparentize($osd_fg_color, 0.7);
+  color: $fg_color;
+  border: 1px solid transparentize(black, 0.9);
+  background-color: transparentize($shell_popup_menu_background_color, 0.5);
```

Comment:
I couldn't figure out how to test the notification count. If this is a separate notification bubble, I suppose it should use basically the same styling as the regular notifications above. Either way, could you by any chance post a screenshot of this?

Reply:
Notification count is overlaid on notification bubble. With the switch to the default notification styling, changed count styling to use transparent black/white to highlight:

![notif-dark](https://user-images.githubusercontent.com/18715287/92301094-e371b400-ef60-11ea-99fa-33f399b76c0d.png)
![notif-lighter](https://user-images.githubusercontent.com/18715287/92301096-e79dd180-ef60-11ea-9ec5-f3bf7f76d395.png)

Reply:
Thanks for the screenshot, for some reason the notifications didn't get grouped with `notify-send`. The notification count, and notifications overall look good for now. Any additional improvements, such as implementing the `.critical` styling can be added later.